kalite/distributed/templates/distributed/base.html

```diff
@@ -187,12 +188,21 @@
                             <!-- End for all users -->
 
                             <!-- For logged in admins -->
-                            {# Teach for admins => assigning playlists #}
-                            <li class="{% block teach_active_admins %}{% endblock teach_active_admins %}">
-                              <a href="{% url 'assign_playlists' %}" id="nav_coachreports" class="admin-only">
-                                {% trans "Teach" %}
-                              </a>
-                            </li>
+                            {% if is_config_package_nalanda %}
```

Comment:
Can we inline the if statement here? It might look cleaner, but I'll leave you to decide on that.